apps/functions/src/functions/enroll.ts

```diff
@@ -105,7 +105,6 @@ export const enroll = Functions.endpoint
             discountCodes,
             paymentMethod: { nonce, deviceData },
             isSignedUpForSolsticeEmails,
-            selectedClassGroups,
```

Comment:
During enrollment, `selectedClassGroups` are no longer collected from the frontend